<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbconfig</class>
 <widget class="QDialog" name="dbconfig">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Подключение к базе данных</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/main/res/db_opened.png</normaloff>:/main/res/db_opened.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="b_exit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_connect">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostname">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="port">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_db_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя бд*</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_username">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя пользователя*</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_port">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>* - поля, обязательные для заполнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_hostname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя хоста</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_state">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
